qt/scientific_interfaces/Inelastic/test/Common/Run/RunPresenterTest.h

```diff
@@ -0,0 +1,87 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include <cxxtest/TestSuite.h>
+#include <gmock/gmock.h>
+
+#include "../MockObjects.h"
+#include "Common/Run/RunPresenter.h"
+#include "MantidQtWidgets/Common/MockUserInputValidator.h"
+
+#include <memory>
```

Comment:
is this include necessary?

Reply:
I think I've included it as its used for `std::unique_ptr` and `std::make_unique`